yarn-project/end-to-end/src/e2e_p2p_network.test.ts

```diff
@@ -77,15 +82,31 @@ describe('e2e_p2p_network', () => {
     for (let i = 0; i < NUM_NODES; i++) {
       const account = privateKeyToAccount(`0x${getPrivateKeyFromIndex(i + 1)!.toString('hex')}`);
       await rollup.write.addValidator([account.address]);
+      logger.debug(`Adding ${account.address} as validator`);
     }
 
+    // Remove the initial sequencer from the set!
```

Comment:
maybe worth noting that it is the sequencer that is explicitly used to perform the setup operations